packages/components/src/ConditionalFilter/GroupFilter.tsx

```diff
@@ -76,7 +111,7 @@ const Group = ({
     };
   }, [isOpen, menuRef]);
 
-  const onToggleClick = (ev) => {
+  const onToggleClick = (ev: { stopPropagation: () => void; persist: () => void }) => {
```

Comment:
I think you should be able to mouse React.MouseEvent. You can re-use the types the `MenuToggle` component is using
```TS
(ev: (JSX attribute) onClick?: React.MouseEventHandler<HTMLButtonElement>)
```